tests/parsers/moldev_softmax_pro/testdata/abs_endpoint_plates.json

```diff
@@ -11,7 +11,7 @@
                     },
                     "measurement document": [
                         {
-                            "measurement identifier": "e3e8807d-1f02-4a4f-80be-5398a7f4235d",
+                            "measurement identifier": "d380f805-450a-4120-a09d-fee2ff7710ac",
```

Comment:
why did all these measurement identifiers change?

Reply:
They change because I update the expected output using a run of the new code. I did this because it was too long to generate by hand, however I went through every change making sure that only the expected values were changed.